pkg/importer/transport.go

```diff
@@ -162,15 +163,9 @@ func processLayer(ctx context.Context,
 				return false, errors.Wrap(err, "Error creating output file's directory")
 			}
 
-			dstFile, err := os.Create(filepath.Join(destDir, hdr.Name))
-			if err != nil {
-				klog.Errorf("Error creating output file: %v", err)
-				return false, errors.Wrap(err, "Error creating output file")
-			}
-
-			if err := copyFile(tarReader, dstFile); err != nil {
```

Comment:
you can probably delete the copyFile function now

Reply:
Of course!